### PR TITLE
Add a scheduled healthcheck to MessageBroker 

### PR DESCRIPTION
**Pull request by naomielst**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -43,6 +43,7 @@ celery = ">= 5.2.0"
 pytest = ">= 6.2.5"
 python = ">= 3.7,<4.0"
 testcontainers = { version = ">= 3.4.2", extras = ["redis"] }
+APScheduler = "^3.8.1"
```
Comment (thedrow) [inline pyproject.toml:46]: In this condition, do we get version 4.x if it is ever released?

File: tests/unit/message_brokers/test_message_broker.py [hunk 2]
```diff
@@ -20,24 +23,61 @@ def container() -> Mock:
...
+    connection_healthy: ConnectionHealthy,
+    disk_space_available: DiskSpaceAvailable,
+):
+    message_broker.check_healthy(connection_healthy, disk_space_available)
+
+    # todo use trigger and minutes constants from MessageBroker
+    call_connection_healthy = call(connection_healthy(), trigger="interval", minutes=1)
+    call_disk_space_available = call(disk_space_available(), trigger="interval", minutes=1)
+
+    healthcheck_scheduler.add_job.assert_has_calls([call_connection_healthy, call_disk_space_available])
```
Comment (thedrow) [inline tests/unit/message_brokers/test_message_broker.py:83]: Good test 👍

File: pyproject.toml (hunk 1 above)
Comment (naomielst) [reply pyproject.toml:46]: We do not. I will change it to >=

File: pyproject.toml [hunk 3]
```diff
@@ -43,7 +43,7 @@ celery = ">= 5.2.0"
 pytest = ">= 6.2.5"
 python = ">= 3.7,<4.0"
 testcontainers = { version = ">= 3.4.2", extras = ["redis"] }
-APScheduler = "^3.8.1"
+APScheduler = ">= 3.8.1"
```
Comment (thedrow) [inline pyproject.toml:46]: Revert this. I was only asking if this is correct.

**Pull request closed without merging**